packages/suite-data/files/translations/en.json

```diff
@@ -400,6 +400,7 @@
   "TR_SWITCH_TO_UNIVERSAL": "Switch to Universal",
   "TR_TAPROOT_ACCOUNTS": "Taproot accounts",
   "TR_TERMS_AND_PRIVACY_CONFIRMATION": "I agree to the <trezor>Trezor Suite Terms</trezor> and <coordinator>zkSNACKs Coordinator Terms</coordinator>",
+  "TR_TOR_DESCRIPTION": "Enabling this will route all of Suite's traffic through the Tor network.{lineBreak}All requests to Trezor infrastructure will be pointed to our Tor services, increasing your privacy and security. Tor may take a while to load and initiate a connection.",
```

Comment:
It is no longer true, that all request to Trezor infrastructure is pointed to our Tor services - we actually remove that redirection completely for now and we use clearnet services via Tor.